### PR TITLE
Create 2023-12-01-survey-launch

### PR DESCRIPTION
**Pull request by graciegregory**
[First draft of the launch blog](https://github.com/rust-lang/blog.rust-lang.org/compare/master...rustfoundation:graciegregory-2023_survey:master#diff-20582b33a5f1e0ba6b9307381262ba56a04c9efc10b8ceb44b7e92d27ba39ad4) for the 2023 State of Rust Survey as discussed between @apiraino, @graciegregory, & @Kobzol in Zulip.

### CODE REVIEW

File: 2023-12-01-survey-launch [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: "Launching the 2023 State of Rust Survey"
+author: The Rust Survey Working Group
+description: "Share your feedback on the eighth year of Rust"
+---
+
+It’s time for the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc)! 
+
+Since 2016, the Rust Project has collected valuable information and feedback from the Rust programming language community through our annual [State of Rust Survey](https://www.surveyhero.com/c/4vxempzc). This tool allows us to more deeply understand how the Rust Project is performing, how we can better serve the global Rust community, and who our community is composed of.
+
+Like last year, the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc) will likely take you between 5 and 20 minutes, and responses are anonymous. We will accept submissions until Friday, December 29th, 2023. Trends and key insights will be shared on [blog.rust-lang.org](https://blog.rust-lang.com) as soon as possible in 2024.
```
Comment (Kobzol) [inline 2023-12-01-survey-launch:12]:
```suggestion
Like last year, the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc) will likely take you between 5 and 20 minutes, and responses are anonymous. We will accept submissions until Friday, December 29th, 2023. Trends and key insights will be shared on [blog.rust-lang.org](https://blog.rust-lang.org) as soon as possible in 2024. 
```

File: 2023-12-01-survey-launch [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+It’s time for the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc)! 
+
+Since 2016, the Rust Project has collected valuable information and feedback from the Rust programming language community through our annual [State of Rust Survey](https://www.surveyhero.com/c/4vxempzc). This tool allows us to more deeply understand how the Rust Project is performing, how we can better serve the global Rust community, and who our community is composed of.
+
+Like last year, the [2023 State of Rust Survey](https://www.surveyhero.com/c/4vxempzc) will likely take you between 5 and 20 minutes, and responses are anonymous. We will accept submissions until Friday, December 29th, 2023. Trends and key insights will be shared on [blog.rust-lang.org](https://blog.rust-lang.com) as soon as possible in 2024. 
+
+We invite you to take this year’s survey whether you have just begun using Rust, you consider yourself an intermediate to advanced user, or you have not yet used Rust but intend to one day. Your responses will help us improve Rust over time by shedding light on gaps to fill in the community, development priorities, and more.
```
Comment (Kobzol) [inline 2023-12-01-survey-launch:14]:
```suggestion
We invite you to take this year’s survey whether you have just begun using Rust, you consider yourself an intermediate to advanced user, or you have not yet used Rust but intend to one day. Your responses will help us improve Rust over time by shedding light on gaps to fill in the community and development priorities, and more.
```

File: 2023-12-01-survey-launch [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+**Once again, we will be offering the State of Rust Survey in the following languages (if you speak multiple languages, please pick one on the main survey page):**
```
Comment (Kobzol) [inline 2023-12-01-survey-launch:16]: The last year's [announcement post](https://blog.rust-lang.org/2022/12/05/survey-launch.html) had actual links to various language version of the survey. Can we again prepare these links?

**Review by Kobzol — COMMENTED** [review]
LGTM in general, although I'm pretty sure we will post it later than 1st of December :) Thanks!

**Review by apiraino — CHANGES_REQUESTED** [review]

File: 2023-12-01-survey-launch (hunk 3 above)
Comment (graciegregory) [reply 2023-12-01-survey-launch:16]: I'm not sure how this was set up last time but I'll ask the team to prep these.

**Comment by Kobzol** [comment]
Thanks for the updates! There are two comments left to be resolved, and also the date should be changed to `2023-12-18`.

File: 2023-12-01-survey-launch [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Launching the 2023 State of Rust Survey"
+author: The Rust Survey Working Group
+description: "Share your feedback on the eighth year of Rust"
```
[the remark this answers is not in the document]
Comment (Kobzol) [reply 2023-12-01-survey-launch:5]: I haven't found any occurrences of this text, neither in text nor in the RSS feed :)

**Pull request closed without merging**